src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -43,6 +43,7 @@ public void GetHostByName_InvalidHostName_Throws(string hostName)
             Assert.ThrowsAny<SocketException>(() => NameResolutionPal.GetHostByName(hostName));
         }
 
+        [ActiveIssue(20245, TestPlatforms.AnyUnix)]
```

Comment:
What about disabling specifially only for OSX10.12? They only seem to fail there.

Or at least only OSX.

```c#
if (PlatformDetection.IsOSX && !IsMacOsHighSierraOrHigher) 
   return;
```

Reply:
I was about to do this ..

Reply:
They've failed on Linux as well.